proto/configpb.proto

```diff
@@ -0,0 +1,98 @@
+syntax = "proto3";
+package configpb;
+
+import "gogoproto/gogo.proto";
+import "rustproto.proto";
+
+option (gogoproto.sizer_all) = true;
+option (gogoproto.marshaler_all) = true;
+option (gogoproto.unmarshaler_all) = true;
+option (rustproto.lite_runtime_all) = true;
+
+service Config {
+    rpc Create(CreateRequest) returns (CreateResponse) {}
+    rpc Get(GetRequest) returns (GetResponse) {}
+    rpc Update(UpdateRequest) returns (UpdateResponse) {}
+}
+
+message Status {
+    enum StatusCode {
+        UNKNOWN = 0;
+        FAILED = 1;
+        STALE_VERSION = 3;
```

Comment:
should be `2`?

Reply:
Should we add variant `NOT_CHANGE` which represents the configuration do not change?